google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/StreamingSubscriberConnection.java

```diff
@@ -165,6 +208,45 @@ private void runShutdown() {
     ackOperationsWaiter.waitComplete();
   }
 
+  private void sendFailureFuture(Throwable t) {
```

Comment:
I guess I am a little confused as to why we would ever need this.  We are creating an empty pubsub message and using it to send an ack or nack failure to the user. In what instance do we want to propagate a failure to a user that is not tied to a real message?

Reply:
Discussed offline and updated.